workspace-hack/build.rs

```diff
@@ -0,0 +1,2 @@
+// A build script is required for cargo to consider build dependencies.
```

Comment:
Shouldn't this also include the license header?

Reply:
It's an empty file that must remain empty. I don't think license header adds any value to that.